news/2021-07-20-project-loved-july-2021.md

```diff
@@ -0,0 +1,267 @@
+
+*Zouka de Arou to Shita Mono* is a lovely 7K beatmap made by Wilben_Chan. This chart consists of comfortable chordstreams of various density and length, scarcely accompanied by long notes. While there isn't anything considerably spectacular about it, this map is a great and smooth experience for everyone who's up for some vanilla chordstreams. The added rates greatly increase the challenge for more adept 7K players, making the set accessible for many skill levels. Considering the above, *Zouka de Arou to Shita Mono* is a great candidate for Project Loved.
+
+[![](/wiki/shared/news/2021-07-20-project-loved-july-2021/648851.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[MAIKA - Goldrop](https://osu.ppy.sh/beatmapsets/648851#mania) by [Destiny19](https://osu.ppy.sh/users/8716749)\
+146 BPM, 3:56 | 8.07★\
+*description written by [Penguinosity](https://osu.ppy.sh/users/10235296)*
+
+Destiny19 makes their first appearance to the Loved votes with their imaginative long note map, *Goldrop*. Destiny is someone that many 7K players are aware of, as they've been putting out some quality maps for about 4 years at this point. It was a tough decision picking what chart from them we would like to see go up first, but after the deliberation we ended on *Goldrop*. This beautiful song found its way into the Loved section some years ago under Skochan's [4K map](https://osu.ppy.sh/beatmapsets/587117), however the experience that Destiny provides is very different.
+
+Starting off pretty slow, players will feel at ease with the slow and comfortable hybrid-esque patterning, but at the 40 second mark Destiny hits you with an unapologetic full inverse stair wall to let you know what's to come. Once the chorus begins, so too do the relentless release-focused LN patterns which also work in some stairs, brackets, and even reverse shields all culminating in a very difficult, but strangely comfortable experience. Rinse and repeat what I've said before until around the 3:30 second mark, where Destiny decides you're getting too comfortable, and slaps you with a full inverse bracket wall that even works in 16th snapped bursts. Although this spike only lasts about 10 seconds, it's one hell of a punch and it's definitely something that players need to watch out for when playing this.
+
+That about does it for all of what I have to say about the chart, it's just a very well-constructed and enjoyable experience, and anybody who enjoys it should definitely go and check out the other maps that Destiny has made if you haven't already. Cast your vote now!
+
+---
+
+From everyone on the [Project Loved Team](/wiki/People/The_Team/Project_Loved_Team), we hope you enjoyed this month's selection, and are looking forward to the next! The favoured beatmaps will hit the [Loved beatmap listing](https://osu.ppy.sh/beatmapsets?s=loved) shortly after the polls end. More info about Project Loved can be found on its [wiki page](/wiki/Project_Loved). See you next month!
```

Comment:
```suggestion
From everyone on the [Project Loved Team](/wiki/People/The_Team/Project_Loved_Team), we hope you enjoyed this month's selection, and are looking forward to the next! The favoured beatmaps will hit the [Loved beatmap listing](https://osu.ppy.sh/beatmapsets?s=loved) shortly after the poll ends. More info about Project Loved can be found on its [wiki page](/wiki/Project_Loved). See you next month!
```

Reply:
This one is a bit ???, since there's more than just 1 poll? This is also how we've worded stuff in the past as well. Personally prefer the wording of the way it is right now.

Reply:
oh, thought there was supposed to be one poll. just ignore this then